packages/patternfly-4/react-core/src/components/Progress/ProgressContainer.js

```diff
@@ -0,0 +1,71 @@
+import React, { Fragment } from 'react';
+import styles from '@patternfly/patternfly-next/components/Progress/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import { CheckCircleIcon, TimesCircleIcon } from '@patternfly/react-icons';
+import ProgressBar from './ProgressBar';
+
+export const ProgressMeasureLocation = {
+  outside: 'outside',
+  inside: 'inside',
+  top: 'top',
+  none: 'none'
+};
+
+export const ProgressVariant = {
+  danger: 'danger',
+  success: 'success',
+  info: 'info'
+};
+
+const propTypes = {
+  /** Progress component DOM ID. */
+  id: PropTypes.string.isRequired,
+  /** Progress title. */
+  title: PropTypes.string,
+  /** Type of progress status. */
+  variant: PropTypes.oneOf(Object.keys(ProgressVariant)),
+  /** Location of progress value. */
+  measureLocation: PropTypes.oneOf(Object.keys(ProgressMeasureLocation)),
+  /** Actual progress value. */
+  value: PropTypes.number.isRequired
+};
+
+const defaultProps = {
+  variant: ProgressVariant.info,
+  measureLocation: ProgressMeasureLocation.Top,
```

Comment:
I think the docs have trouble evaluating this. Could just use the string as a workaround or we try to fix this in react-docs